_posts/2018-02-28-foreman-community-newsletter---february-2018.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+title: Foreman Community Newsletter - February 2018
+date: 2018-02-28 12:43:00
+author: Ori Rabin
+tags:
+- foreman
+- newsletter
+- community
+excerpt: |
+  1.17 RC1, Katello 3.5.1 & 3.6 RC1 and Lots of Deep dives!
+---
+
+Thank you to everyone that came to see us and especially those that gave
+talks in FOSDEM and Config Management Camp!
+
+### Foreman 1.17 RC1 out
+
+The Foreman first Release Candidate for 1.17 was released two weeks ago.
+This version finally has the vertical navigation you've seen in
+the last few demos. See the [announcement][1.17-rc1] for installation details.
+
+As ever, we'd be glad to get feedback and appreciate all the help with
+testing and fixes! Please do continue to log the [bugs][issues] you
+find in order to make this release better.
+
+### Katello 3.5.1 released
+
+Katello 3.5.1 is here! This version works with Foreman 1.16 and provides
+features and bug fixes all around - installer, content views, API and more.
+More details in the [announcement][k_3_5_1]. Please do [log][k_issues] any issues
+you find with this.
+
+
+### Katello 3.6 RC1 out
+
+Katello 3.6 is available for testing! This version will work with Foreman 1.17.
+This brings in a lot of features and bug fixes which can be seen in the
+[change log][k_3_6_rc1]. Reporting of [bugs][k_issues] will be appreciated.
+
+### Community survey 2018
+
+The survey is still open. We have some nice responses and we want more!
+Let's beat last years responses and help us shape the direction of the
+project.
+
+http://goo.gl/RNY8MY
+
+### Upcoming Events
+
+* Community Demo - 8th March (URL to be announced)
+
+Time again for the latest round-up of action from the community, covering core,
+proxies, plugins, and more.  As always, you're welcome to participate in the
+show, either via YouTube Live chat, or via our IRC channel
+(Freenode/#theforeman).
+
+* [Deep Dive - Foreman-Probing - 7th March](https://www.youtube.com/watch?v=-PYhFH7YraU)
+In this deep dive we'll meet a new plugin, foreman-probing.
+This plugin allows scanning networks and creating Hosts in Foreman for
+the detected machines. Adam will describe the plugin's features, inner
+working and possible usage for brownfield deployments.
+
+* [Deep Dive - Telemetry in Foreman - 14th March](https://www.youtube.com/watch?v=QoJ-r8YfWEI)
+Rails internal telemetry was recently added into Foreman Core, in this demo
+Lukáš Zapletal is going to configure and show how to integrate Foreman with Prometheus.
+The next part will cover all currently exported metrics, how to read them and utilize
+for finding bottlenecks. In the last part, he is gonna show how to add your own
```

Comment:
`gonna` - `going to show` ?

Reply:
Copied from the video description added by Lukas :) I'll change here and the description.